Source/FakeItEasy.Specs/LazySpecs.cs

```diff
@@ -50,5 +70,12 @@ protected override IFoo Create()
                 return instance;
             }
         }
+
+        public class Bar
+        {
+            internal Bar()

```

Comment:
I know this is really picky, specially in a test lib, but we generally avoid `internal` on members and only use it on entire types.